config/ruby.yml

```diff
@@ -0,0 +1,23 @@
+default: &default
+  versions:
+    - version: 3.1
+      default: true
```

Comment:
I'd take `default` out of `versions` to escape multiple `default` (occasionally).

And probably add a check to including `default` in `versions`.

Reply:
Is there any reason to not just have the first version in the list be the default?

Reply:
> Is there any reason to not just have the first version in the list be the default?

Not sure. Probably if we'll have some beta or `head` version docs. But until not — it's an option too.

Reply:
There already is. It's called "dev" and it's the last version in this YAML file.

Reply:
> There already is. It's called "dev" and it's the last version in this YAML file.

Oh, I'm sorry. In this case, I'm not sure that we should place it as the latest one (excluding obsolete). Like… it's obviously more than latest (3.1 for now), but it's under 2.6 and above 2.5.